projects/dagster-slurm/pyproject.toml

```diff
@@ -28,6 +28,38 @@ Documentation = "https://ascii-supply-networks.github.io/dagster-slurm/docs/intr
 [tool.uv.build-backend]
 module-root = ""
 
+[tool.pixi.dependencies]
```

Comment:
should we assume these are globally installed instead or wrap them into a dedicated buld environment?